dist/index.js

```diff
@@ -2388,7 +2388,6 @@ const fs = __importStar(__webpack_require__(747));
 const path = __importStar(__webpack_require__(622));
 const ARCH = process.env.ARCH || 'linux';
 const githubToken = core.getInput('github-token');
-core.info(githubToken);
```

Comment:
I'm surprised to see the dist folder - shouldn't this be built locally on each machine/ci/etc .. using the repo ?

Reply:
The way the action works is it calls the code from the dist file (see `action.yml` line 24)

Reply:
Okay - this could be reworked I think (see https://docs.github.com/en/enterprise-cloud@latest/actions/creating-actions/sharing-actions-and-workflows-with-your-enterprise) but not necessary for now.